broadcast/internal/arc/mocks/arc_client_mock_success_nill_query_transaction_response.go

```diff
@@ -0,0 +1,56 @@
+package mocks
+
+import (
+	"context"
+
+	broadcast_api "github.com/bitcoin-sv/go-broadcast-client/broadcast"
+	"github.com/bitcoin-sv/go-broadcast-client/broadcast/broadcast-client-mock/fixtures"
+)
+
+type ArcClientMockNilQueryTransactionResponse struct{}
+
+// GetFeeQuote returns a successful FeeQuote response.
+func (*ArcClientMockNilQueryTransactionResponse) GetFeeQuote(ctx context.Context) ([]*broadcast_api.FeeQuote, error) {
+	quotes := make([]*broadcast_api.FeeQuote, 0)
+	quotes = append(quotes, Fee1)
+	quotes = append(quotes, Fee2)
```

Comment:
u could add this in one line `quotes := []*broadcast_api.FeeQuote{Fee1, Fee2}`

Reply:
Reduced to one line